docs/DeveloperGuide.adoc

```diff
@@ -782,13 +782,19 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
 
 *Target user profile*:
 
-* has a need to manage a significant number of contacts
+* promotes and sells products or services through social media
```

Comment:
Maybe add a one-line description of what is your target user. All things listed here are the characteristics of your user.